snippets/visualbasic/VS_Snippets_Network/RequestDataUsingWebRequest/vb/WebRequestGetExample.vb

```diff
@@ -0,0 +1,31 @@
+Imports System  
+Imports System.IO  
+Imports System.Net  
+Imports System.Text  
+
+Namespace Examples.System.Net  
+    Public Class WebRequestGetExample  
+        Public Shared Sub Main()  
+            ' Create a request for the URL.   
+            Dim request As WebRequest = _  
+              WebRequest.Create("http://www.contoso.com/default.html")  
+            ' If required by the server, set the credentials.  
+            request.Credentials = CredentialCache.DefaultCredentials  
+            ' Get the response.  
+            Dim response As WebResponse = request.GetResponse()  
+            ' Display the status.  
+            Console.WriteLine(CType(response,HttpWebResponse).StatusDescription)  
+            ' Get the stream containing content returned by the server.  
+            Dim dataStream As Stream = response.GetResponseStream()  
+            ' Open the stream using a StreamReader for easy access.  
+            Dim reader As New StreamReader(dataStream)  
```

Comment:
This should be wrapped in a using statement and the call to reader.Close removed.